assembl/static2/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "build": "node --max-old-space-size=8192 ./node_modules/.bin/webpack --progress --colors",
+    "build": "webpack --progress --colors",
```

Comment:
You should probably add the `node --max-old-space-size=8192 ./node_modules/.bin/` back in here - can kill a machine until we move to Terser

Reply:
Roger that !

Reply:
@gary-bluenove Ok I thought rebooting the server would prevent adding that option. In fact, building with the `max-old-space-size` break the build stage. Without that option build stage is green.

Reply:
ok leave it off - we've added to underpowered machine but if you don't need, you don't need. It builds locally and on CI, we're good